spring-cloud-task-core/src/main/java/org/springframework/cloud/task/listener/TaskLifecycleListener.java

```diff
@@ -92,18 +95,26 @@
 	@Value("${spring.cloud.task.closecontext.enable:true}")
 	private Boolean closeContext;
 
+	@Value("${spring.cloud.task.executionid:}")
+	private Integer taskExecutionId;
+
+	@Value("${spring.cloud.task.external.executionid:#{null}}")
```

Comment:
There are also no tests that use this property.  We need to test the injection works.  Also, is the `null` really needed?